src/app/common/hooks/analytics/use-analytics.ts

```diff
@@ -62,7 +62,7 @@ export function useAnalytics() {
         const [category, name, properties, options, ...rest] = args;
         const prop = { ...defaultProperties, ...properties };
         const opts = { ...defaultOptions, ...options };
-        logger.debug(`Analytics page view: ${name}`, properties);
+        // logger.debug(`Analytics page view: ${name}`, properties);
```

Comment:
Get rid of this maybe?

Reply:
https://github.com/pinojs/pino/issues/1861